Dockerfile

```diff
@@ -62,7 +62,6 @@ RUN apt-get update && \
     docker-compose-plugin
 
 
-COPY . /experiment
-WORKDIR /experiment
-RUN /venv/bin/pip install --disable-pip-version-check -r requirements.txt
-ENTRYPOINT ["./report/docker_run.sh"]
+CMD git clone --depth=1 --branch=main https://github.com/google/oss-fuzz-gen.git /experiment && \
```

Comment:
I don't think we should be going down this path tbh... Having the clone at on docker run startup is fragile and creates weird issues when we want to customize the entry point with different run timeout or experiment types.

I had a chat with Dongge about why he thinks https://github.com/google/oss-fuzz-gen/issues/42 is necessary and it comes down to making sure the images for daily experiments are built from a clean state.

I'm adding a github action to build a new image after every commit on main (https://github.com/google/oss-fuzz-gen/pull/49) and I think that's a better path to make sure the daily experiments are representative of the repo at HEAD.

Reply:
Agreed, github action feels more elegant than what we proposed here. I'll close this pr.